k8s/examples/elasticsearch/README.md

```diff
@@ -0,0 +1,59 @@
+
+# Elasticsearch with StorageOS
+
+Elasticsearch is a distributed, RESTful search and analytics engine, most popularly used to aggregate logs, but also to serve as search backend to a number of different applications.
+
+This repository is part of our use-case documentation and provides example YAML manifests to help you get started on running [Elasticsearch with StorageOS](https://docs.storageos.com/docs/usecases/kubernetes/elasticsearch).
+
+Please visit the official docs page and please do leave us feedback for improvement or simply open a PR.
+
+## Note
+
+This example deployment requires a certain amount of resources as it will be deploying
+
+- 3x data nodes
+- 3x coordinator nodes
+- 3x master nodes
+
+requiring a combined ~14GB of memory (minimum), however, more memory may be
+used by the application/s.
+
+# Pre-Requisites
```

Comment:
I suggest the use of init containers to set the `max_map_count`. This prereq creates friction and makes the app crash right away if not met. 
I applied the changes in the PR https://github.com/storageos/deploy/pull/53/files to reflect the init container in the manifests.